cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceUtils.java

```diff
@@ -65,4 +74,84 @@ public static KafkaRebalanceState rebalanceState(KafkaRebalanceStatus kafkaRebal
         }
         return null;
     }
+
+    /**
+     * Compose the name of the KafkaRebalance custom resource to be used for running an auto-rebalancing in the specified mode
+     * for the specified Kafka cluster
+     *
+     * @param cluster   Kafka cluster name (from Kafka custom resource metadata)
+     * @param kafkaRebalanceMode    Auto-rebalance mode
+     * @return  the name of the KafkaRebalance custom resource to be used for running an auto-rebalancing
+     *          in the specified mode for the specified Kafka cluster
+     */
+    public static String autoRebalancingKafkaRebalanceResourceName(String cluster, KafkaRebalanceMode kafkaRebalanceMode) {
+        return cluster + "-auto-rebalancing-" + kafkaRebalanceMode.toValue();
+    }
```

Comment:
I  know the `KafkaResources` class is a bit mess. But it is a resource naming method belonging to the Kafka cluster. Would it make sense to have it there?

Reply:
Yeah right, we have that class. Moving there.